safe_transaction_service/history/tasks.py

```diff
@@ -233,17 +233,18 @@ def process_decoded_internal_txs_for_safe_task(
                             logger.error(message)
                             index_service = IndexServiceProvider()
                             if previous_safe_status := safe_status.previous():
-                                block_number = (
-                                    previous_safe_status.internal_tx.ethereum_tx.block_id
-                                )
+                                block_number = previous_safe_status.block_number
```

Comment:
Just to confirm: `previous_safe_status.internal_tx.ethereum_tx.block_id` is equivalent to `previous_safe_status.block_number` (ie.: the respective block numbers should match)?

Reply:
Yes, that's only a shortcut